src/services/formatting.js

```diff
@@ -10,6 +10,20 @@ import { formats } from '../constants';
 const nullOrUndefined = value => _.isNull(value) || _.isUndefined(value);
 const toBigNumber = value => new BigNumber(String(value));
 
+const formatNumberBase = (value, wholeNumberFormat) => {
+  let bigNumber = toBigNumber(value);
+  const isNegative = bigNumber.isNegative();
+  bigNumber = bigNumber.abs();
+  let wholeNumber = bigNumber.integerValue(BigNumber.ROUND_FLOOR);
+  const fractionalNumber = bigNumber.minus(wholeNumber);
+  if (!wholeNumber.isZero()) {
+    wholeNumber = isNegative ? wholeNumber.multipliedBy(-1) : wholeNumber;
+    return `${numeral(wholeNumber).format(wholeNumberFormat)}`
+      + `${numeral(fractionalNumber).format(formats.FRACTIONAL_NUMBER)}`;
+  }
+  return (isNegative ? '-0' : '0') + numeral(fractionalNumber).format(formats.FRACTIONAL_NUMBER);
+};
```

Comment:
I moved this logic up top to match APH-Wallet-GUI. It makes sense since I need to use this logic in another method down below.